segmentation/training_ResNeXt50_for_Semantic_Segmentation_on_SYNTHIA.ipynb

```diff
@@ -408,8 +408,7 @@
    "source": [
     "Data augmentation is a powerful technique to increase the amount of your data and prevent model overfitting.  \n",
     "If you not familiar with such trick read some of these articles:\n",
-    " - [The Effectiveness of Data Augmentation in Image Classification using Deep\n",
-    "Learning](http://cs231n.stanford.edu/reports/2017/pdfs/300.pdf)\n",
+    " - [The Effectiveness of Data Augmentation in Image Classification using Deep Learning](http://cs231n.stanford.edu/reports/2017/pdfs/300.pdf)\n",
     " - [Data Augmentation | How to use Deep Learning when you have Limited Data](https://medium.com/nanonets/how-to-use-deep-learning-when-you-have-limited-data-part-2-data-augmentation-c26971dc8ced)\n",
     " - [Data Augmentation Experimentation](https://towardsdatascience.com/data-augmentation-experimentation-3e274504f04b)\n",
     "\n",
```

Comment:
Can we just delete everything between these lines, so we don't have to worry about links breaking in the future?